common/persistence/cassandraPersistence.go

```diff
@@ -2804,6 +2928,28 @@ func createSignalInfo(result map[string]interface{}) *SignalInfo {
 	return info
 }
 
+func createBufferedReplicationTaskInfo(result map[string]interface{}) *BufferedReplicationTask {
+	info := &BufferedReplicationTask{}
+	for k, v := range result {
+		switch k {
+		case "first_event_id":
+			info.FirstEventID = v.(int64)
+		case "next_event_id":
+			info.NextEventID = v.(int64)
+		case "version":
+			info.Version = v.(int64)
+		case "history":
+			h := v.(map[string]interface{})
+			info.History = createSerializedHistoryEventBatch(h)
+		case "new_run_history":
```

Comment:
what is this new run history?

Reply:
Replication Task for the ContinueAsNew has the history for both current run and new run as the same transaction.